src/plugins/query_enhancements/public/query_assist/components/query_assist_summary.tsx

```diff
@@ -63,11 +63,17 @@ export const convertResult = (body: IDataFrame) => {
   return hits;
 };
 
+enum FeedbackStatus {
```

Comment:
do you think we should add this to common types?

https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/src/plugins/query_enhancements/common/query_assist/types.ts#L21